src/apps/Organization/Tokens.js

```diff
@@ -0,0 +1,277 @@
+    </div>
+  </NoTokensCard>
+)
+
+const TokenDetails = ({ token }) => {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const [removeDialogOpen, setRemoveDialogOpen] = useState(false)
+  const oneColumn = layoutName === 'small'
+  const toggleRemoveDialog = () => setRemoveDialogOpen(!removeDialogOpen)
+
+  return (
+    <React.Fragment>
+      <Modal visible={removeDialogOpen} onClose={toggleRemoveDialog}>
+        <Text
+          size="xlarge"
+          css={`
+            margin: 20px 0 26px 0;
+            display: block;
+          `}
+        >
+          Remove token
+        </Text>
+        <Text size="small">
+          This action will remove your token from the organization, until you
+          decide to reconnect it.
+        </Text>
+        <div
+          css={`
+            display: flex;
+            justify-content: flex-end;
+            margin-top: 20px;
+          `}
+        >
+          <Button compact mode="outline" onClick={toggleRemoveDialog}>
+            Cancel
+          </Button>
+          <Button
+            compact
+            mode="strong"
+            css={`
+              background: ${theme.negative};
+              margin-left: 10px;
+            `}
+            onClick={toggleRemoveDialog}
+          >
+            Remove token
+          </Button>
+        </div>
+      </Modal>
+
+      <TokenDetailsCard
+        css={`
+          padding: 20px;
+          display: flex;
+          flex-direction: ${oneColumn ? 'column' : 'row'};
+        `}
+      >
+        <div
+          css={`
+            width: ${oneColumn ? '100' : '49'}%;
+            margin-bottom: ${oneColumn ? '20px' : '0'};
+          `}
+        >
+          <div css="display: flex">
+            <div css="margin-bottom: 20px; width: 50%">
+              <Label text="Name" />
+              <Text.Block>{token.tokenName}</Text.Block>
+            </div>
+            <div css="margin-bottom: 20px">
+              <Label text="Symbol" />
+              <Text.Block>{token.tokenSymbol}</Text.Block>
+            </div>
+          </div>
+
+          <div css="margin-bottom: 20px">
+            <Label text="Address" block />
+            <LocalIdentityBadge
+              entity={token.tokenAddress}
+              shorten={layoutName !== 'large'}
+            />
+          </div>
+
+          <div css="margin-bottom: 20px">
+            <Label text="Description" block />
+            <TextInput.Multiline wide>
+              {token.tokenDescription}
+            </TextInput.Multiline>
+          </div>
+
+          <div>
+            <Button mode="strong" css="margin-right: 10px">
+              Save changes
+            </Button>
+            <Button onClick={toggleRemoveDialog} mode="outline">
+              Remove token
+            </Button>
+          </div>
+        </div>
+
+        <div
+          css={`
+            width: ${oneColumn ? '100' : '50'}%;
+            padding-left: ${oneColumn ? '0' : '20px'};
```

Comment:
There are definitely ways to accomplish all of this with CSS; no need to rely on JS to tell us if it's one column. I'm not sure if it's worth the effort for now, but it might be a good way to learn some CSS together, @rkzel.